CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - [#2012](https://github.com/JabRef/jabref/issues/2012) Implemented integrity check for `month` field: Should be an integer or normalized (BibLaTeX), Should be normalized (BibTeX)
 
 ### Fixed
+- Improved error messages when using fetcher
```

Comment:
That's up for debate. Before the error messages didn't work as intended (or didn't work at all).